src/Dispatcher.php

```diff
@@ -1,12 +1,13 @@
 <?php
+declare(strict_types=1);
 
 namespace FastRoute;
 
 interface Dispatcher
 {
-    const NOT_FOUND = 0;
-    const FOUND = 1;
-    const METHOD_NOT_ALLOWED = 2;
+    public const NOT_FOUND          = 0;
+    public const FOUND              = 1;
+    public const METHOD_NOT_ALLOWED = 2;
```

Comment:
I'm not sure that alignment by longest const is a good idea. If const will be renamed or newly added will be longer, then others should be edited too. But technically they not changed.

Reply:
@dmytro-protsyk nice catch, thanks! That behaviour can be configured (ignore whitespaces on diff). However, I've removed the sniff in the configuration file (because the project didn't use this style before) and forgot to apply it here.